resources/metadata/transportRequestUploadCTS.yaml

```
@@ -72,45 +74,51 @@ spec:
       - name: abapPackage
         type: string
         mandatory: true
-        description: "The ABAP package name of your application"
+        description: "ABAP package name of the UI5 application"
         scope:
           - PARAMETERS
           - STAGES
           - STEPS
           - GENERAL
       - name: osDeployUser
         type: string
-        default: node
-        description: "By default we use a standard node docker image and prepare some fiori related packages before performing the deployment. For that we need to launch the image with root privileges. After that, before actually performing the deployment we swith to a non root user. This user can be specified here."
+        default: 'node'
+        description: "Docker image user performing the deployment"
         aliases:
+          - name: changeManagement/osDeployUser
           - name: changeManagement/cts/osDeployUser
         scope:
           - PARAMETERS
           - STAGES
           - STEPS
           - GENERAL
       - name: deployConfigFile
-        default: ui5-deploy.yaml
+        default: 'ui5-deploy.yaml'
         type: string
         aliases:
+          - name: changeManagement/deployConfigFile
           - name: changeManagement/cts/deployConfigFile
-          - name: cts/deployConfigFile
-        description: "The ABAP package name of your application"
+        description: "Configuration file for the fiori deployment"
         scope:
           - PARAMETERS
           - STAGES
           - STEPS
           - GENERAL
       - name: transportRequestId
+        resourceRef:
+          - name: commonPipelineEnvironment
+            param: custom/transportRequestId
         type: string
         mandatory: true
-        description: "The id of the transport request to upload the file. This parameter is only taken into account when provided via signature to the step."
+        description: "ID of the transport request to which the UI5 application is uploaded"
         scope:
           - PARAMETERS
       - name: deployToolDependencies
         type: "[]string"
-        description: "By default we use a standard node docker iamge and prepare some fiori related packages performing the deployment. The additional dependencies can be provided here. In case you use an already prepared docker image which contains the required dependencies, the empty list can be provide here. Caused hereby installing additional dependencies will be skipped."
+        default: ['@ui5/cli', '@sap/ux-ui5-tooling', '@ui5/logger', '@ui5/fs']
+        description: "List of additional dependencies to fiori related packages needed to install on a standard node docker image. Provide an empty list, in case your docker image already contains the required dependencies. Caused hereby installing additional dependencies will be skipped"
```

```suggestion
        description: "List of additional dependencies to fiori related packages needed to install on a standard node docker image. Provide an empty list in case your docker image already contains the required dependencies. Caused hereby installing additional dependencies will be skipped"
```
I don't understand the last sentence.

---

how do think about
```
        description: "List of additional dependencies to fiori related packages. 
           By default a standard node docker image is used on which the dependencies are installed. 
           Provide an empty list, in case your docker image already contains the required dependencies"

```

---

Yes, that sounds good.